eth/filters/api.go

```diff
@@ -270,6 +271,36 @@ func (api *PublicFilterAPI) PendingBlock(ctx context.Context) (*rpc.Subscription
 	return rpcSub, nil
 }
 
+// reOrg sends a notification each time a new pending block is created.
+func (api *PublicFilterAPI) ReOrg(ctx context.Context) (*rpc.Subscription, error) {
+	notifier, supported := rpc.NotifierFromContext(ctx)
+	if !supported {
+		return &rpc.Subscription{}, rpc.ErrNotificationsUnsupported
+	}
+
+	rpcSub := notifier.CreateSubscription()
+
+	go func() {
+		reOrg := make(chan core.ReOrgRollup)
+		reOrgSub := api.events.SubscribeReOrg(reOrg)
+
+		for {
+			select {
+			case b := <-reOrg:
+				notifier.Notify(rpcSub.ID, b)
```

Comment:
What is rpcSub.ID?

Reply:
Every time someone subscribes we create a sub ID and send it as a first response. Similar to other implementations in that file.

Reply:
That is the identifier for the subscription what the unsubscribe method also uses to close the subscription